src/EventListener/BuildAttributeListener.php

```diff
@@ -54,11 +54,12 @@ public function buildAttribute(BuildAttributeEvent $event)
             $this->addAttributeToDefinition($container, $name);
             $properties->getProperty($name . '__sort')->setWidgetType('fileTreeOrder');
 
+            $properties->addProperty($property = new DefaultProperty($name . '__sort'));
```

Comment:
This is wrong. As you see in the lines above, the property is already added and therefore we only change the widget type then.
Your change adds a new property with the same name and therefore replaces the existing property.

Can you elaborate on the original problem?

Reply:
For an attribute of type file, if we did not enable `Multiple selection` then it works fine  because of:
```php
 if (!($attribute instanceof File) || !$attribute->get('file_multiple')) {
       return;
 }
```


But when we enable `Multiple selection`, we have a double widget creation.  In the original file (BuildAttributeListener.php), on line 60,
```php
$properties->addProperty($property = new DefaultProperty($name . '__sort'));
```
This creates an empty default property and sets its type to 'fileTreeOrder'. This is new property doesn't have `files`,`extensions`, `path`,  `etc`,  So I think it is wrong to create a new property.

Maybe my first pull request wasn't correct. Ideally should the function `buildAttribute` not be like this? 

```php
    public function buildAttribute(BuildAttributeEvent $event)
    {
        $attribute = $event->getAttribute();
        if (!($attribute instanceof File) || !$attribute->get('file_multiple')) {
            return;
        }

        $container  = $event->getContainer();

        $properties = $container->getPropertiesDefinition();
     
        $name       = $attribute->getColName();

        if ($properties->hasProperty($name)) {

            $properties->getProperty($name)->setWidgetType('fileTreeOrder');

            return;
        }
    }
```




May be I am still wrong or not seeing a complete picture as I don't understand 
```
$nameSort   = \sprintf('%s__sort', $name);
```
what this line is for...